frontend/lib/src/components/elements/Toast/styled-components.tsx

```diff
@@ -67,4 +67,7 @@ export const StyledMessageWrapper = styled.div(({ theme }) => ({
   minHeight: "100%",
   fontSize: theme.fontSizes.sm,
   lineHeight: theme.lineHeights.base,
+  div: {
```

Comment:
Previously, this modification was part of the Streamlit markdown. But since it is toast specific, I moved it back here.